src/main.rs

```diff
@@ -35,7 +35,14 @@ fn main() {
         .short("p")
         .long("path")
         .value_name("PATH")
-        .help("The path that the prompt should render for")
+        .help("The filesystem path that the prompt should render for")
+        .takes_value(true);
+
+    let logical_path_arg = Arg::with_name("logical-path")
+        .short("P")
+        .long("logical-path")
+        .value_name("LOGICALPATH")
+        .help("The shell path that the prompt should render for")
```

Comment:
**suggestion:**
Please update the help string to distinguish between `path` and `logical-path`.